api/schema/v1/modules/packetio.yaml

```diff
@@ -361,6 +367,49 @@ definitions:
                 description: Client identifier
         required:
           - method
+      ipv6:
+        type: object
+        description: IPv6 protocol configuration
+        properties:
+          method:
+            type: string
+            description: Configuration method
+            enum:
+              - static
+              - dhcp6
+              - auto
+          link_local_address:
+            type: string
+            description: IPv6 link local address
+          static:
+            type: object
+            description: Static configuration parameters
+            properties:
+              address:
+                type: string
+                description: IPv6 address
+              prefix_length:
+                type: integer
+                description: Prefix length
+                format: int32
+              gateway:
+                type: string
+                description: Default gateway
```

Comment:
IPv6 prefix length and gateway options are not supported by lwip.
Prefix lengths are always /64 and gateway is not supported because it only supports routes/prefixes from NDP.